### ISSUE TITLE
A potential bug in nni/nas/pytorch/mutator, line no. 87

### ISSUE THREAD

**Issue by Davidxswang** [microsoft/nni#3525]
In nni/nas/pytorch/mutator, line no.87, a `.tolist()` is missing to convert the numpy array into a list, as shown below:

```
def status(self):
    """
    Return current selection status of mutator.

    Returns
    -------
    dict
        A mapping from key of mutables to decisions. All weights (boolean type and float type)
        are converted into real number values. Numpy arrays and tensors are converted into list.
    """
    data = dict()
    for k, v in self._cache.items():
        if torch.is_tensor(v):
            v = v.detach().cpu().numpy()       # here, a .tolist() is missing
        if isinstance(v, np.ndarray):
            v = v.astype(np.float32).tolist()
        data[k] = v
    return data
```

**Comment by ultmaster** [comment]
Sorry for the late response. Please submit a PR directly to fix it!

**Comment by Davidxswang** [comment]
> Sorry for the late response. Please submit a PR directly to fix it!

@ultmaster I have submitted a PR. https://github.com/microsoft/nni/pull/3561
Thanks!

**Comment by Davidxswang** [comment]
I'll close it since this has been fixed in version 2.2.
